myhpom/templates/myhpom/accounts/signup.html

```diff
@@ -20,68 +20,95 @@ <h5 class="pseudo-modal__title--centered">Already have an account? <a href="{% u
-                            <label for="id_first_name">First name</label>
-                            <input type="text" class="form-control" name="first_name" id="id_first_name" />
+                            <label for="id_first_name">{{form.fields.first_name.label|upper}}</label>
+                            <input type="text" class="form-control{% if form.errors.first_name %} is-invalid{% endif %}" name="first_name" id="id_first_name" value="{% if form.data.first_name %}{{form.data.first_name}}{% endif %}" />
+                            <div class="invalid-feedback">
+                                {% if form.errors.first_name %}{{ form.errors.first_name }}{% endif %}
+                            </div>
                         </div>
                         <div class="col-md-6 col-lg-4 form-group">
-                            <label for="id_middle_name">Middle name (optional)</label>
-                            <input type="text" class="form-control" name="middle_name" id="id_middle_name" />
+                            <label for="id_middle_name">{{form.fields.middle_name.label|upper}} (Optional)</label>
+                            <input type="text" class="form-control{% if form.errors.middle_name %} is-invalid{% endif %}" name="middle_name" id="id_middle_name" value="{% if form.data.middle_name %}{{form.data.middle_name}}{% endif %}" />
+                            <div class="invalid-feedback">
+                                {% if form.errors.middle_name %}{{ form.errors.middle_name }}{% endif %}
+                            </div>
                         </div>
                         <div class="col-md-6 col-lg-4 form-group">
-                            <label for="id_last_name">Last name</label>
-                            <input type="text" class="form-control" name="last_name" id="id_last_name" />
+                            <label for="id_last_name">{{form.fields.last_name.label|upper}}</label>
+                            <input type="text" class="form-control{% if form.errors.last_name %} is-invalid{% endif %}" name="last_name" id="id_last_name" value="{% if form.data.last_name %}{{form.data.last_name}}{% endif %}" />
+                            <div class="invalid-feedback">
+                                {% if form.errors.last_name %}{{ form.errors.last_name }}{% endif %}
+                            </div>
                         </div>
                     </div>
                     <div class="row">
                         <div class="col-lg-8">
                             <div class="row">
                                 <div class="col-md-6 form-group">
-                                    <label for="id_email_address">Email address</label>
-                                    <input type="email" class="form-control is-invalid" name="email_address" id="id_email_address" />
+                                    <label for="id_email">{{form.fields.email.label|upper}}</label>
```

Comment:
The wireframes for error validation show that the corresponding labels should also be styled in red, which isn't happening here.

Reply:
The other relevant missing style here is the terms and conditions box, and title of the terms and conditions.